src/main/webapp/user/UserRepresentation.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * This class is responsible for displaying an icon for the creator of
+ * placeguides, used in each placeguide info box. The icon creates a link to
+ * the portfolio of the given user.
+ */
+class UserRepresentation {
+  constructor(user) {
+    this._userRepresentationDiv = UserRepresentation.createUserDiv(user);
+  }
+
+  get div() {
+    return this._userRepresentationDiv;
+  }
+
+  static createUserDiv(user) {
+    const userDiv = document.createElement("div");
+    userDiv.style.height = "5px";
+    userDiv.style.width = "50px";
```

Comment:
Shouldn't this be part of some "userDiv" CSS class?

Reply:
Sorry, this part remained there from a previous implementation, it is not needed anymore, I removed it completelt.